bin/run-consumer

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+demo_dir="$( cd "$(dirname "$0")/.." ; pwd -P )"
+consumer_id="consumer"
```

Comment:
I don't like this being hard-coded in a script in `bin` - it ought to be an env var, `DEMO_CONSUMER_ID` or something like that. Then we can make changes in the docker-compose file and have them propagate through the demo.